boards/arm/degu_evk/degu_evk.dts

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright (c) 2019 Atmark Techno, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52840_qiaa.dtsi>
+
+/ {
+	model = "Degu Evaluation Kit";
+	compatible = "nordic,pca10056-dk", "nordic,nrf52840-qiaa",
+		     "nordic,nrf52840";
+
+	chosen {
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		led0: led_0 {
+			gpios = <&gpio1 7 GPIO_INT_ACTIVE_LOW>;
+			label = "LED1";
+		};
+		led1: led_1 {
+			gpios = <&gpio1 5 GPIO_INT_ACTIVE_LOW>;
+			label = "LED2";
+		};
+		led2: led_2 {
+			gpios = <&gpio1 3 GPIO_INT_ACTIVE_LOW>;
+			label = "LED3";
+		};
+		led3: led_3 {
+			gpios = <&gpio1 4 GPIO_INT_ACTIVE_LOW>;
+			label = "LED4";
+		};
+	};
+
+	buttons {
+		compatible = "gpio-keys";
+		button0: button_0 {
+			gpios = <&gpio1 0 GPIO_PUD_PULL_UP>;
+			label = "SW1";
```

Comment:
This button is labelled as "SW2" on the [schematic](https://github.com/open-degu/hardware/blob/master/Degu_Baseboard_schematic_rev1.1.pdf) and in the [Hardware Overview](https://github.com/open-degu/hardware), "SW1" is RESET. Wouldn't it be good to match those labels?

Reply:
Thank you!
In our old schematic, these labels ware shifted SW1 to SW3 and I forgot that.
I will fix it.